x-pack/test/observability_functional/apps/observability/alerts/rules_page.ts

```diff
@@ -0,0 +1,176 @@
+      await esArchiver.unload('x-pack/test/functional/es_archives/observability/alerts');
+      await esArchiver.unload('x-pack/test/functional/es_archives/infra/metrics_and_logs');
+    });
+
+    describe('Feature flag', () => {
+      // Related to the config inside x-pack/test/observability_functional/with_rac_write.config.ts
+      it('Link point to O11y Rules pages by default or when "xpack.observability.unsafe.rules.enabled: true"', async () => {
+        const manageRulesPageHref = await observability.alerts.rulesPage.getManageRulesPageHref();
+        expect(new URL(manageRulesPageHref).pathname).equal('/app/observability/alerts/rules');
+      });
+    });
+
+    describe('Create rule button', () => {
+      it('Show Create Rule flyout when Create Rule button is clicked', async () => {
+        await observability.alerts.common.navigateToRulesPage();
+        await retry.waitFor(
+          'Create Rule button is visible',
+          async () => await testSubjects.exists('createRuleButton')
+        );
+        await observability.alerts.rulesPage.clickCreateRuleButton();
+        await retry.waitFor(
+          'Create Rule flyout is visible',
+          async () => await testSubjects.exists('addRuleFlyoutTitle')
+        );
+      });
+    });
+
+    describe('Rules table', () => {
+      let uptimeRuleId: string;
+      let logThresholdRuleId: string;
+      before(async () => {
+        const uptimeRule = {
+          params: {
+            search: '',
+            numTimes: 5,
+            timerangeUnit: 'm',
+            timerangeCount: 15,
+            shouldCheckStatus: true,
+            shouldCheckAvailability: true,
+            availability: { range: 30, rangeUnit: 'd', threshold: '99' },
+          },
+          consumer: 'alerts',
+          schedule: { interval: '1m' },
+          tags: [],
+          name: 'uptime',
+          rule_type_id: 'xpack.uptime.alerts.monitorStatus',
+          notify_when: 'onActionGroupChange',
+          actions: [],
+        };
+        const logThresholdRule = {
+          params: {
+            timeSize: 5,
+            timeUnit: 'm',
+            count: { value: 75, comparator: 'more than' },
+            criteria: [{ field: 'log.level', comparator: 'equals', value: 'error' }],
+          },
+          consumer: 'alerts',
+          schedule: { interval: '1m' },
+          tags: [],
+          name: 'error-log',
+          rule_type_id: 'logs.alert.document.count',
+          notify_when: 'onActionGroupChange',
+          actions: [],
+        };
+        uptimeRuleId = await createRule(uptimeRule);
+        logThresholdRuleId = await createRule(logThresholdRule);
+        await observability.alerts.common.navigateToRulesPage();
+      });
+      after(async () => {
+        await deleteRuleById(uptimeRuleId);
+        await deleteRuleById(logThresholdRuleId);
+      });
+
+      it('shows the rules table ', async () => {
+        await testSubjects.existOrFail('rulesList');
```

Comment:
This _seems_ to fix the failures I saw while testing locally;
```
       it('shows the rules table ', async () => {
         await testSubjects.existOrFail('rulesList');
+        await testSubjects.waitForDeleted('centerJustifiedSpinner');
         const tableRows = await find.allByCssSelector('.euiTableRow');
         const rows = await getRulesList(tableRows);
         expect(rows.length).to.be(2);
```

Reply:
I added it